tests/cpp/test_resize.cpp

```diff
@@ -23,7 +23,7 @@
 
 namespace nvfuser {
 
-using ResizeTest = NVFuserTest;
+using ResizeTest = NVFuserFixtureParamTest<bool>;
```

Comment:
Pad unit tests run both the legacy and new indexers. Will change the slice and cat tests later.